packages/react-scripts/config/paths.js

```diff
@@ -48,6 +48,10 @@ function getServedPath(appPackageJson) {
   return ensureSlash(servedUrl, true);
 }
 
+const sWPrecacheImportScript = fs.existsSync(resolveApp('public/service-worker-import.js'))
+  ? 'service-worker-import.js'
```

Comment:
Build script [is creating](https://github.com/facebookincubator/create-react-app/blob/v1.0.10/packages/react-scripts/config/webpack.config.prod.js#L324) a `service-worker.js` file - that's why I decided to use `service-worker-import.js`.

I guess there is such naming convention for files which are served from public directory.

Unlike the `testsSetup.js` which is located in `src` dir, this one should be placed in `public` and is just copied over as it is into `build` by build script.